Docs/Tools/compileStats.sh

```diff
@@ -2,5 +2,4 @@
 
 ./fetchStats.sh
 ./csvPerChange.sh
-python csvGrapher.py 
-
+python3 csvGrapher.py
```

Comment:
On my system there's only `python2` and `python3`. Do you have `python3` also? If yes, then this should work.